wagn/lib/wagn/application.rb

```diff
@@ -90,17 +94,12 @@ def paths
         paths['app/mailers'] = []
         
         add_gem_path paths, "lib/tasks",     :with => "lib/wagn/tasks", :glob => "**/*.rake"
-        add_gem_path paths, 'gem-assets'
-        add_gem_path paths, 'config/routes', :with => 'lib/wagn/config/routes.rb'
+        add_gem_path paths, 'config/routes', :with => 'rails/application-routes.rb'
```

Comment:
Interesting.  I like that better, what suggested it to you?

And, are you putting gem-assets with decko-rails, gem-wise?  I think that's right, it should be in a different place for initializing the path.